spec/migrations/20160203101130_add_orderable_to_orchestration_templates_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_migration
+
+describe AddOrderableToOrchestrationTemplates do
+  let(:orchestration_template_stub) { migration_stub(:OrchestrationTemplate) }
+
+  migration_context :up do
+    it "sets true to column orderable" do
+      orchestration_template_stub.create!
+      orchestration_template_stub.create!
```

Comment:
Same as above... It's not obvious to me why we need to create two of these instances here.  It feels like two identical assertions.